playbooks/common/openshift-cluster/openshift-hosted/cockpit-ui.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Create Hosted Resources - cockpit-ui
+  hosts: oo_first_master
+  pre_tasks:
+  - name: Scrape openshift_master_public_api_url
+    include: master_facts_tasks.yml
+    when: openshift_master_public_api_url is not defined
+  roles:
+  - role: cockpit-ui
+    when: ( openshift.common.version_gte_3_3_or_1_3  | bool ) and ( openshift_hosted_manage_registry | default(true) | bool ) and not (openshift.docker.hosted_registry_insecure | default(false) | bool)
```

Comment:
Not required, but with ``and`` logic it's nice to have it multiline. Something like:

```yaml
    when: 
    - ( openshift.common.version_gte_3_3_or_1_3  | bool ) 
    - ( openshift_hosted_manage_registry | default(true) | bool ) 
    - not (openshift.docker.hosted_registry_insecure | default(false) | bool)
```

Reply:
These are copy-pasta from openshift_hosted.yml

Reply:
Understood. I don't see any of my nits to be required.